woodwork/datatable.py

```diff
@@ -1000,7 +1000,7 @@ def _make_categorical_for_mutual_info(self, data, num_bins):
             data[col_name] = new_col.cat.codes
         return data
 
-    def mutual_information_dict(self, num_bins=10, nrows=None):
+    def mutual_information_dict(self, num_bins=10, nrows=None, include_index=False):
```

Comment:
Assumed we wanted to default to not including the index, but happy to change

Reply:
I'll defer to @gsheni on this point.

Reply:
We should default to False for now.